docs/ref/checks.txt

```diff
@@ -97,6 +97,8 @@ Fields
 * **fields.W161**: Fixed default value provided.
 * **fields.W900**: ``IPAddressField`` has been deprecated. Support for it
```

Comment:
I'll probably keep it, too. Just as historical documentation. You should put `E900` before `W900` though.